server/controllers/entities/lib/create_wd_entity.ts

```diff
@@ -1,17 +1,20 @@
-import { getClaimValue } from '#controllers/entities/lib/inv_claims_utils'
+import { getClaimValue, getFirstClaimValue } from '#controllers/entities/lib/inv_claims_utils'
 import { getWikidataOAuthCredentials, validateWikidataOAuth } from '#controllers/entities/lib/wikidata_oauth'
 import { newError } from '#lib/error/error'
-import { mapKeysValues, objectEntries } from '#lib/utils/base'
+import { arrayIncludes, mapKeysValues, objectEntries } from '#lib/utils/base'
+import { requireJson } from '#lib/utils/json'
 import { log } from '#lib/utils/logs'
 import { relocateQualifierProperties } from '#lib/wikidata/data_model_adapter'
 import wdEdit from '#lib/wikidata/edit'
-import type { EntityUri, EntityValue, ExpandedClaims, InvExpandedPropertyClaims, InvSnakValue, Labels, PropertyUri, Reference, ReferenceProperty, ReferencePropertySnaks, WdEntityId, WdEntityUri, WdPropertyId } from '#server/types/entity'
+import type { EntityUri, EntityValue, ExpandedClaims, InvExpandedPropertyClaims, InvSnakValue, Labels, PropertyUri, Reference, ReferenceProperty, ReferencePropertySnaks, WdEntityId, WdEntityUri, WdPropertyId, InvClaimObject } from '#server/types/entity'
 import type { User } from '#server/types/user'
 import { getInvEntityType } from './get_entity_type.js'
 import { prefixifyWd, unprefixify } from './prefix.js'
 import { getPropertyDatatype } from './properties/properties_values_constraints.js'
 import { validateInvEntity } from './validate_entity.js'
-import type { SimplifiedQualifiers } from 'wikibase-sdk'
+import type { SimplifiedQualifiers, WikimediaLanguageCode } from 'wikibase-sdk'
+
+const wmLanguageCodeByWdId = requireJson('wikidata-lang/mappings/wm_code_by_wd_id.json')
 
 const allowlistedEntityTypes = [ 'work', 'serie', 'human', 'publisher', 'collection' ]
```

Comment:
If this PR also intends to enable moving an edition to wikidata therefore:

```suggestion
const allowlistedEntityTypes = [ 'work', 'serie', 'human', 'publisher', 'collection', 'edition' ]
```

But then, there is also some work to tranfer the image `inv:P2`, and there might be some work to handdle the case where the isbn value is already used by a wd item. 

So maybe the feature "move an edition to wikidata" could be done in a separate PR (?)